changelogs/unreleased/6526-index-float-int.yml

```
@@ -0,0 +1,9 @@
+description: number type should no longer differentiate between integer and decimal notation when used in indexes.
+issue-nr: 6526
+issue-repo: inmanta-core
+change-type: minor
+destination-branches: [master]
+sections:
+  feature: "Introduce the 'float' type which is like a float in python"
+  deprecation-note: "Deprecate the 'number' type. Instead the 'int' or 'float' type should be used"
```

I think we should try to avoid passive voice
```suggestion
  deprecation-note: "Deprecate the 'number' type. Use the 'int' or 'float' type instead"
```